SEED-platform/seed
# Remove nosetests and fix precommit

#### Any background context you want to provide?
Nosetests have been out of development for several years and no longer a preferred solution for django testing. I also needed to run a specific test with debug within VSCode, which wasn't supported by nosetest (at least no easily).

#### What's this PR do?
* removes the django-nose package and related code
* update the pytest.ini to support running with existing db
* fix the name of the normalize address test
* update flake8 pre-commit to point to github (was gitlab) and ran `pre-commit autoupdate` to get the latest versions

#### How should this be manually tested?
* verify that `./manage.py test` still works as it used to
* test using `pytest` method or just `pytest --collect-only` to save time


![image](https://user-images.githubusercontent.com/1907354/202045348-e1dedf1f-b530-45a3-a05a-bae4dd2918a5.png)

* test running a test or debug in VSCode (choose `pytest` as the testing framework with VSCode)
![image](https://user-images.githubusercontent.com/1907354/202045360-43fe73f2-72fb-472e-b1c8-ce44e612a11c.png)

#### What are the relevant tickets?

#3617
